### PR TITLE
feat: release 0.1.0 improvements-2

### PR DESCRIPTION
**Pull request by heeth-arda**
## Description

- Updated docs
- Remove calling out any names (it wasn't intentional, and by accident)
- Fixed simulation error response (better response now)
- Update issue templates (minor changes)

**Issue Reference:** #14

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality/improvements/enhancements)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] I have followed the guidelines in our [Contributing](../CONTRIBUTING.md) document
- [x] I have checked that there aren't other open [Pull Requests](../../../pulls) for the same update/change
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (if any)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,18 +16,39 @@ Arda - Risk Management for digital assets
...
 - [License](#license)
 
 ## Description
 
-Safeguard allows users to make informed decisions before signing a transaction. Users can understand the risks involved in a transaction such as phishing attacks, wallet drainers, rug pulls, malicious domains, and much more
+### What is Olynthus?
+
+- Olynthus (called Safeguard in the codebase) is an open source project by Arda which can be used to provide transaction insights for your on chain activity in real time and help you safeguard against loosing out on the assets.
+
+### What do we need Safeguard?
```
Comment (rishisundar) [inline README.md:39]:
```suggestion
### Why do we need Safeguard?
```
Also, should we use Safeguard here? Wouldn't this confuse the users with too many codewords?

File: README.md [hunk 2]
```diff
@@ -16,18 +16,39 @@ Arda - Risk Management for digital assets
...
+
+- Let us assume a user wants to transfer 100 USD Coin to another address (X) via a web app / smart contract
+  The web app / smart contract would generate a request which the user must confirm to proceed
+- If the web app /smart contract is trustworthy (source code is publicly available and is audited), we can be rest assured that it will indeed transfer 100 USD Coin to address X
+- But, if the web app / smart contract is malicious and the source code is not publicly available (like for most of the smart contracts available on the blockchain), it might show the request as transferring 100 USD Coin to X, but in reality, it might transfer all of our USD Coin to their own address (Y) and drain us of our tokens
+
+### How does Safeguard solves the problem?
```
Comment (rishisundar) [inline README.md:46]:
```suggestion
### How does Safeguard solve the above problem?
```

File: README.md (hunk 1 above)
Comment (heeth-arda) [reply README.md:40]: yes, made changes to use Safeguard everywhere instead

File: README.md [hunk 3]
```diff
@@ -32,9 +32,10 @@ Arda - Risk Management for digital assets
...
 ## Description
 
-### What is Olynthus?
+### What is Safeguard (code name Olynthus)?
 
-- Olynthus (called Safeguard in the codebase) is an open source project by Arda which can be used to provide transaction insights for your on chain activity in real time and help you safeguard against loosing out on the assets.
+- Olynthus is the codebase name for the project, Safeguard.
+- Safeguard is an open source project by Arda which can be used to provide transaction insights for your on chain activity in real time and help you safeguard against loosing out on the assets.
 
 ### What do we need Safeguard?
```
Comment (rishisundar) [inline README.md:40]:
```suggestion
### Why do we need Safeguard?
```

**Review by rishisundar — DISMISSED** [review]

**Review by rishisundar — APPROVED** [review]

**Pull request merged**